go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openshift-kni/numaresources-operator
 
-go 1.17
+go 1.18
```

Comment:
you may want to pull RTE 0.6.0

Reply:
Ah, thanks for cutting a release on the RTE side!

Reply:
Will update this PR now.